packages/db/src/types/file.ts

```diff
@@ -0,0 +1,19 @@
+export interface PostgresFile {
+  granule_cumulus_id: number
+  bucket?: string
+  checksum_type?: string
+  checksum_value?: string
+  filename?: string
+  file_name?: string
```

Comment:
Just curious, why do we have `filename` and `file_name`?

Reply:
I think the short answer is that the duplicate column names are leftover from the Dynamo implementation.

The fuller explanation starts with the fact that in the past, we decided (for reasons I cannot recall) to change some of the property names for storing files in DynamoDB:

https://github.com/nasa/cumulus/pull/840/files?file-filters%5B%5D=.js#diff-c35b25b91a802220b991bbe398342ab5131967deba67ff184243e0a32271ccb8

However, we didn't want to take on the work of migrating all existing DynamoDB file data to the "new" schema, so we wrote the changes to be backwards compatible. The unfortunate side effect of that approach was that we had to permanently support both `fileName` and `filename` properties in DynamoDB (and other clear duplicates) and still do to this day. 

And in an attempt to limit the scope of the RDS work, we decided not to take on any work to drop/rename columns, so we initially defined the RDS file schema to include `fileName` and `filename` properties. But then when we renamed columns to snake case we changed `fileName` to `file_name`:

https://github.com/nasa/cumulus/pull/1984/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts#diff-e5107b2d790fca17fbcace381c4059eba860c21e9419d51be27852919d18e3b3

However, given that I have renamed some columns for collections (`granuleIdExtraction` in Dynamo -> `granule_id_extraction_regex` in RDS) and the fact that change to snake case is itself a material change to the column names, I think I might vote in favor of removing these duplicates in RDS now. Even if we do that, we can still easily translate an RDS record with only a `file_name` back to the Dynamo/API format with `fileName`/`filename` .

@Jkovarik @nemreid - Thoughts?

Thanks for forcing me to look more closely at this @botanical

Reply:
After team discussion, we agreed that we can remove the duplicate column names for the Postgres schemas and keep only the snake case column names, but this will be done in a separate PR